CONTRIBUTING.md

```diff
@@ -72,6 +72,20 @@ If you are modifying one or more of the NPM packages in the `/packages` director
 GitHub provides additional document on [forking a repository](https://help.github.com/articles/fork-a-repo/) and
 [creating a pull request](https://help.github.com/articles/creating-a-pull-request/).
 
+## Keeping code generation in sync with aws-sdk-js-v3
+
+Any changes made to the `smithy-typescript-codegen` package need to be compatible with aws-sdk-js-v3. Maintainers and reviewers
+MUST ensure that code generation is kept in sync between the two repositories by creating an equivalent PR in aws-sdk-js-v3.
+
+Procedure to keep code generation in sync while making changes to `smithy-typescript-codegen`:
+1. Fork and clone [aws/aws-sdk-js-v3][aws-sdk-js-v3].
+2. Run `yarn` to install dependencies.
+3. Run `generate-clients` with `HEAD` commit as follows:
+    ```sh
+    yarn generate-clients --commit HEAD
```

Comment:
mention that this commit refers to smithy-typescript, and not JSv3?

This `HEAD` with unspecified repo implicitly requires that you have smithy-typescript in the same parent folder as JSv3, right?

Reply:
> mention that this commit refers to smithy-typescript, and not JSv3?

It's mentioned in the help section for generate-clients script.

> This `HEAD` with unspecified repo implicitly requires that you have smithy-typescript in the same parent folder as JSv3, right?

Actually, the `repo` option might be confusing as it refers to location where smithy-typescript repository is downloaded and not the name of the repository. We can update it, if required.

The `generate-clients` script clones the smithy-typescript repository in default location (which is sibling folder of aws-sdk-js-v3) if it's not already present and deletes it after client creation is complete. It's not required to be present.